src/main/java/be/seeseemelk/mockbukkit/Coordinate.java

```diff
@@ -31,6 +31,16 @@ public Coordinate()
 		this(0, 0, 0);
 	}
 
+	/**
+	 * Gets the chunk coordinate this coordinate is in.
+	 *
+	 * @return The chunk coordinate of this coordinate.
+	 */
+	public ChunkCoordinate toChunkCoordinate()
+	{
+		return new ChunkCoordinate(x / 16, z / 16);
```

Comment:
This should rather use bit-shifting.
```suggestion
		return new ChunkCoordinate(x >> 4, z >> 4);
```